modules/config/home-cursor.nix

```diff
@@ -42,7 +42,8 @@ let
 
       gtk = {
         enable = mkEnableOption ''
-          gtk config generation for <option>home.pointerCursor</option>
+          gtk config generation for <option>home.pointerCursor</option>.
+	  Note that <option>gtk</option> has to be enabled for the configs to take effect
```

Comment:
It may be worthwhile to add a clause under the main `home.pointerCursor` submodule's description since the same requirement holds for the x11 backend and `xsession.enable`.

Perhaps something akin to
```nix
description = ''
  Cursor configuration. Set to <literal>null</literal> to disable.
  </para><para>
  Top-level options declared under this submodule are backend independent
  options. Options declared under namespaces such as <literal>x11</literal>
  are backend specific options. By default, only backend independent cursor
  configurations are generated. If you need configurations for specific
  backends, you can toggle them via the enable option. For example,
  <xref linkend="opt-home.pointerCursor.x11.enable"/>
  will enable x11 cursor configurations.
  </para><para>
  Note that the cursor configurations are only generated, the relevant
  subsytems must be configured for them to take effect. For example,
  <xref linkend="opt-home.pointerCursor.gtk.enable"/> will enable
  gtk cursor configuration but <xref linkend="opt-gtk.enable"/> needs
  to be set for it to be applied.
'';
```

Reply:
Good point. I've now removed the note on the gtk option and added the more general note under the main submodule description :)